content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,241 @@
+
+#### Configure precise code navigation for 9 languages in a matter of minutes with auto-indexing
+Code navigation allows you to quickly jump to symbol definitions, find references, dependencies, and more, so you can spend more time shipping code. Precise code navigation is the most powerful version of code navigation and it works cross-repository. 
+
+Auto-indexing allows you to set up precise code navigation for any chosen repositories with the click of a button, and it automatically keeps your repositories indexed and your [code graph data](https://docs.sourcegraph.com/code_navigation/explanations/uploads) up-to-date for accurate code navigation. With the release of 4.0, auto-indexing is now supported for 9 languages including Ruby, Rust, Go, Java, Scala, Kotlin, Python, TypeScript, JavaScript, and it's turned on by default for our Cloud customers.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Auto-Indexing%20and%20Code%20Navigation',
+    mp4: 'blog/release-post/4.0/Auto-Indexing%20and%20Code%20Navigation'
+  }}
+  loop={true}
+  title="Ruby precise code navigation"
+  caption="Auto-indexing is now supported for 9 languages and allows you to set up precise code navigation, automatically"
+/>
+
+<a href="https://docs.sourcegraph.com/code_navigation/references/indexers" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<a href="https://docs.sourcegraph.com/integration"><Badge text="Extensions" color="vermillion" size="small" /></a>
+
+#### Your favorite extensions are now available by default 
+Top used extensions, including code navigation, git-extras, open-in-editor, and search-exports, are now part of the platform and no longer require you to  enable on your own. Essential code context is surfaced during the ideal moments in your workflow and knowledge about the codebase from the extensions can be found in the sidebar, when hovering over code, and after clicking on "more actions."
+
+Now that our top extensions are part of the core functionality, there will be greater support and functionality over time. With this change, you will no longer be able to access the extensions registry or create extensions on a private registry. Note: if you still need access to extensions or create extensions on a private registry, you can enable a feature flag to do so until early 2023.
+
+This update *does not* impact our IDE extensions, which will continue to allow you to search and navigate across all of your repositories without ever leaving your IDE or checking them out locally.
+
+<div className="tw-mb-sm">
+  <Figure
+    src="https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Blame.png"
+    alt="Git Blame"
+  />
+</div>
+
+<br />
+<a href="/code-search"><Badge text="Code Search" color="cerise" size="small" /></a>
+
+#### Quickly access answers within your codebase with a revamped reference panel 
```

Comment:
@macraig you please review this by EOD today to ensure the content is accurate? Please keep in mind that we're focusing on the benefits and value to our customers/prospects within this release post.

Reply:
LGTM, my only nitpick is that "quickly access answers" sounds a bit weird. "Quickly get answers"? or "Find answers faster"?

Reply:
Thanks! Updated to Find answers faster with a revamped referenced panel